src/sdk/bucketspace_fs.js

```diff
@@ -231,7 +231,14 @@ class BucketSpaceFS extends BucketSpaceSimpleFS {
                 return;
             }
             const bucket_name = this.get_bucket_name(entry.name);
-            const bucket = await object_sdk.read_bucket_sdk_config_info(bucket_name);
+            let bucket;
+            // in case the config file was deleted during the list - we will continue (silent)
+            try {
+                bucket = await object_sdk.read_bucket_sdk_config_info(bucket_name);
+            } catch (err) {
+                if (err.code !== 'ENOENT') throw err;
```

Comment:
@shirady I think that on this case we need to not fail on anything, WDYT?

Reply:
Can you add dbg.warn() error message?

Reply:
Do you mean that you do not want to fail any error when you try to list the buckets using the API?
I can add, but it will be very verbose in case many buckets were deleted concurrently.

Reply:
Right, but it'll help debug
I meant to remove this ENOENT and always skip,
we need to think about what's worse, failing the whole list op on (for example) non accessible bucket.json or not showing the bucket on the list, it's a dilemma because not showing the bucket is also not good
NOT NOW - on NooBaa CLI we can add an error section to the bucket on the list, but on S3 it's a defined protocol, so I suppose that omitting the bucket and add a warning on the logs is the correct thing to do.
@guymguym

Reply:
The decision for this PR is to add `dbg.warn` and continue if the bucket does not exist.
The check will be on the `rpc_code` of `NO_SUCH_BUCKET` because that is the translated error from `read_bucket_sdk_config_info`, and we want to be precise on the specific case.